### PR TITLE
feat: merge scheduler into orchestrator

### PR DESCRIPTION
**Pull request by wangzhuzhen**
#### What this PR does / why we need it:
Merge scheduler into orchestrator

#### Specified Reviewers:

/assign @sixther-dc @iutx @luobily

#### ChangeLog
Feature: Merge scheduler into orchestrator（将 scheduler 模块合并到 orchestrator 模块中）

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |       Merge scheduler into orchestrator     |
| 🇨🇳 中文    |       将 scheduler 模块合并到 orchestrator 模块中       |

### CODE REVIEW

File: .erda/migrations/pipeline/20220215-pipeline-base-update.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+UPDATE `pipeline_configs`
```
Comment (iutx) [inline .erda/migrations/pipeline/20220215-pipeline-base-update.sql:18]: This migration is not need，dcos executor is deleted.

File: modules/openapi/api/apis/scheduler/scheduler_nodelabel_list.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 var SCHEDULER_NODELABEL_LIST = apis.ApiSpec{
```
Comment (iutx) [inline modules/openapi/api/apis/scheduler/scheduler_nodelabel_list.go:22]: move to apis/orchestrator better, and rename variable to OCHESTRATOR_NODELABEL_LIST

File: .erda/migrations/pipeline/20220215-pipeline-base-update.sql (hunk 1 above)
Comment (wangzhuzhen) [reply .erda/migrations/pipeline/20220215-pipeline-base-update.sql]: delete this file

File: modules/openapi/api/apis/scheduler/scheduler_nodelabel_list.go (hunk 2 above)
Comment (wangzhuzhen) [reply modules/openapi/api/apis/scheduler/scheduler_nodelabel_list.go]: moved those files

**Review by sixther-dc — APPROVED** [review]
/approve

**Pull request merged**